agentMET4FOF/metrological_streams.py

```diff
@@ -60,12 +62,18 @@ def __init__(
             standard uncertainties associated with values
         time_unc : iterable of floats or float, optional (defaults to 0)
             standard uncertainties associated with timestamps
+        exp_unc : iterable of floats or float, optional (defaults to None)
+            expanded uncertainties associated with values
+        cov_factor : iterable of integers or integer, optional (defaults to 1)
+            coverage factor associated with the expanded uncertainty
```

Comment:
These type specifications should match the type hints. As of now they diverge since in the docstring we still offer iterables.

Reply:
Additionally I would suggest to leave out the specification of default values for `exp_unc` and `cov_factor`, since the default values basically mean "no effect", which is what I would expect of I can optionally provide these parameters but choose to not do this. `defaults to None` is pretty technical and may be beyond the scope of interest of some users. If we want to be more informative, would it maybe add helpful information if we stated something like

> if left out, plain `time_unc` and `value_unc` are used, otherwise `value_unc` is computed from it

and maybe even provide the formula (although it is very simple)?

Reply:
This is outdated.